README.md

```diff
@@ -3,7 +3,7 @@
 Renders UML diagram as defined in a text file. For full syntax, see:
 http://plantuml.sourceforge.net/index.html
 
-1. Install the extension from Chrome Web Store.
+1. Install the [extension](https://chrome.google.com/webstore/detail/plantuml-viewer/legbfeljfbjgfifnkmpoajgpgejojooj?hl=en) from Chrome Web Store.
```

Comment:
This won't work, we need the patched version from this repo

Reply:
ohh I didn't realize. thanks.